script.js

```diff
@@ -0,0 +1,41 @@
+const result = document.querySelector('#app')
+const ul = document.getElementById('#results')
+
+result.addEventListener('submit', () => {
+    fetch('http://localhost:3000/results')
+    .then(resp => resp.json())
+    .then(data => data.forEach((item) => {
+        const li = document.createElement('li')
+        li.textContent = item.fact
+        console.log(li)
+        ul.appendChild(li)
+    }))
+})
+
```

Comment:
So I understand what you went for, firstly I dont think forEach would work on the data after the second then chain. We approached this differently, you want to make sure your event listener is a 'click' instead of submit. You can also use better naming conventions for your variables as its sometimes confusing whats linked to what.